interpreter/Makefile

```diff
@@ -182,27 +83,8 @@ dunetest:
 
 .PHONY:		clean
 
-$(ZIP):		$(WINMAKE)
-		git archive --format=zip --prefix=$(NAME)/ -o $@ HEAD
+$(ZIP):
+	git archive --format=zip --prefix=wasm/ -o $@ HEAD
 
 clean:
-		rm -rf _build/jslib $(LIB).mlpack _tags $(JSLIB).js
-		$(OCB) -clean
-
-
-# Opam support
```

Comment:
Why did you remove the opam support?

Reply:
It's broken currently. I'll need to make a few fixes to get it back (e.g. move the `dune-project` at the root of the repository again, see [ocaml/dune#7962](https://github.com/ocaml/dune/issues/7962). Currently it crashes, when it'll be fixed, we'll still get an error saying that dependencies outside of workspace is not supported. You can reproduce easily by doing `dune runtest` from the `interpreter` directory on current main.

Reply:
I made a minimal opam support.